### ISSUE TITLE
Generate BrowserCluster jobs from --scope-extend-paths with fragments

### ISSUE THREAD

**Issue by Zapotek** [Arachni/arachni#776]
Currently, `--scope-extend-paths` are normalized and only fetched by the `HTTP::Client`, thus removing fragments, which can be used to restore DOM states.
Instead, pass paths with fragments as `BrowserCluster` jobs.

**Comment by bykes** [comment]
+1

**Comment by lukaszbadziak** [comment]
+1
Especially support for # in URLs really welcome, especially for modern application based on dynamic solutions, e.g. Angular

**Comment by nitrocode** [comment]
Yes this is definitely needed and seems to be lacking in other automated web application testing.

**Comment by m6a-UdS** [comment]
+1
